### ISSUE TITLE
usage of system collections

### ISSUE THREAD

**Issue by pekeler** [arangodb-foxx/demo-sessions#1]
Following this example, new users are stored in the system collection `_users` which is also used internally by Arango to store the database user. A database user is something very different than a user within the context of my application, and storing both together in the same collection seems fundamentally wrong.

Similarly for sessions: if a session gets created by a user of my app, why would it be considered a system `_session`?

If I have an app "myapp" which requires users and persistent sessions, I'd expect those to be stored in `myapp_users`, `myapp_sessions`, in addition to all my other `myapp_*` collections.

Or am I misunderstanding something?

It seems that if I mount the foxx-users app under /users, it creates a non-system `users_users` collection. But I can't quite wrap my head around how to make my app work with the mounted foxx-users app.

**Comment by pekeler** [comment]
So mounting the app foxx-users under /users, and changing https://github.com/arangodb/foxx-sessions-example/blob/ad31f85158f844a763eb9655dd0d90444bb837dc/index.js#L11 to

``` JavaScript
var users = Foxx.requireApp('/users').userStorage;
```

seems to do the trick in that the `users_users` collection is used instead of `_users`. Is this the correct and intended way of doing this?

**Comment by pluma** [comment]
Yes. The users app creates a collection under its namespace when you mount it and exposes an API that uses that collection.

I need to update the users and session apps as well as this example app now that the system users and session apps are separate from the ones available in the app store.
